scripts/install.sh

```diff
@@ -0,0 +1,20 @@
+npm install \
+  flat@"^4.1.0" \
```

Comment:
is it necessary to add dependencies here?

Reply:
Yes, it is necessary. AFAIK, when package is installed in a project, it will only install package dependencies, `peerDependencies` and `devDependencies` not installed. Also the dependencies will be installed in the project `node_modules`. Once installed the package won't have it's necessary dependencies to do build task.